cli/tests/integration/lsp_tests.rs

```diff
@@ -1933,12 +1935,13 @@ fn lsp_hover_unstable_disabled() {
       "contents": [
         {
           "language": "typescript",
-          "value": "any"
-        }
+          "value": "type Deno.ForeignLibraryInterface = /*unresolved*/ any",
```

Comment:
Ugh, so this test basically just stopped testing what's important here due to `Deno.ppid` stabilization?

Reply:
Yeah, though the test above it will fail so for next time I left a comment there pointing to this one.